images/macos/scripts/build/configure-machine.sh

```diff
@@ -50,21 +50,18 @@ done
 rm -f ./add-certificate
 
 # enable-automationmode-without-authentication
-
+brew install expect
```

Comment:
Would this tool appear in SBOM? If not then we need to make sure it's removed from the image

Reply:
Got it. Removing from the image.